python/GafferArnoldTest/ArnoldRenderTest.py

```diff
@@ -1534,5 +1534,127 @@ def testInstancerEncapsulatePerf( self ) :
 			with GafferTest.TestRunner.PerformanceScope() :
 				s["render"]["task"].execute()
 
+	@GafferTest.TestRunner.PerformanceTestMethod( repeat = 1 )
+	def testInstancerManyPrototypesPerf( self ) :
+
+		s = Gaffer.ScriptNode()
+
+		s["plane"] = GafferScene.Plane()
+		s["plane"]["divisions"].setValue( imath.V2i( 500 ) )
+
+		s["sphere"] = GafferScene.Sphere()
+		s["sphere"]["expression"] = Gaffer.Expression()
+
+		s["pathFilter"] = GafferScene.PathFilter()
+		s["pathFilter"]["paths"].setValue( IECore.StringVectorData( [ '/plane' ] ) )
+
+		s["instancer"] = GafferScene.Instancer()
+		s["instancer"]["in"].setInput( s["plane"]["out"] )
+		s["instancer"]["filter"].setInput( s["pathFilter"]["out"] )
+		s["instancer"]["prototypes"].setInput( s["sphere"]["out"] )
+
+		s["instancer"]["contextVariables"].addChild( GafferScene.Instancer.ContextVariablePlug( "context" ) )
+		s["instancer"]["contextVariables"][0]["name"].setValue( "P" )
+		s["instancer"]["contextVariables"][0]["quantize"].setValue( 0 )
+
+		s["render"] = GafferArnold.ArnoldRender()
+		s["render"]["in"].setInput( s["instancer"]["out"] )
+
+		with Gaffer.Context() as c :
+			c["scene:render:sceneTranslationOnly"] = IECore.BoolData( True )
+			with GafferTest.TestRunner.PerformanceScope() :
+				s["render"]["task"].execute()
+
+	@GafferTest.TestRunner.PerformanceTestMethod( repeat = 1 )
+	def testInstancerManyPrototypesEncapsulatePerf( self ) :
+
+		s = Gaffer.ScriptNode()
+
+		s["plane"] = GafferScene.Plane()
+		s["plane"]["divisions"].setValue( imath.V2i( 500 ) )
+
+		s["sphere"] = GafferScene.Sphere()
+		s["sphere"]["expression"] = Gaffer.Expression()
+
+		s["pathFilter"] = GafferScene.PathFilter()
+		s["pathFilter"]["paths"].setValue( IECore.StringVectorData( [ '/plane' ] ) )
+
+		s["instancer"] = GafferScene.Instancer()
+		s["instancer"]["in"].setInput( s["plane"]["out"] )
+		s["instancer"]["filter"].setInput( s["pathFilter"]["out"] )
+		s["instancer"]["prototypes"].setInput( s["sphere"]["out"] )
+
+		s["instancer"]["contextVariables"].addChild( GafferScene.Instancer.ContextVariablePlug( "context" ) )
+		s["instancer"]["contextVariables"][0]["name"].setValue( "P" )
+		s["instancer"]["contextVariables"][0]["quantize"].setValue( 0 )
```

Comment:
This test isn't setting `encapsulateInstanceGroups` either, so I'm a bit confused about what we're trying to test. It also has the unused `s["sphere"]["expression"]`.

Reply:
I don't think I've changed anything on this commit yet, but it is setting encapsulateInstanceGroups - right below this comment on line 1627 ... though I'm not sure this will still be visible once I update the PR, since I've done some squashing.